src/modules/buses/controller/busesController.ts

```diff
@@ -0,0 +1,90 @@
+import { Request, Response } from 'express';
+ import busesRepository from '../repository/busesRepository';
+
+ import { INTERNAL_SERVER_ERROR, OK } from 'http-status'
+ import responseUtil from '../../../utils/responseUtil'
+
+ const createBus = async (req: Request, res: Response)=> {
+  try {
+    const bus = await busesRepository.createBus(req.body)
+    responseUtil.handleSuccess(OK, 'Success', bus)
+    return responseUtil.response(res)
+  } catch (err:any) {
+    responseUtil.handleError(INTERNAL_SERVER_ERROR, err.toString())
+    return responseUtil.response(res)
+  }
+};
+
+export const deleteBus = async (req: Request, res: Response) =>{
+    try {
+      const id=parseInt(req.params.id)
+        const data = await busesRepository.deleteBus(id);
+        responseUtil.handleSuccess(OK, 'Success', data);
+        return responseUtil.response(res);
+    } catch (error: any) {
+        responseUtil.handleError(INTERNAL_SERVER_ERROR, error.toString());
+        return responseUtil.response(res);
+    }
+}
+
+export const findBus = async (req: Request, res: Response) => {
+  try {
+    const bus = await busesRepository.getABus(req.params.id);
+
+    if (!bus) {
+      return res.status(404).json({
+        status: "fail",
+        message: "Bus with that ID not found",
+      });
+    }
+
+    res.status(200).json({
+      status: "success",
+      data: {
+        bus,
+      },
```

Comment:
I think here you were supposed to use the general response format that we are all using 
for example for the first response
` responseUtil.handleError(BAD_REQUEST, `Bus with that ID not found`);
            return responseUtil.response(res);`